tests/data/events/register_response.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You put the file in the events folder, but in the test you're searching for it in the correct folder.

Reply:
moved where it goes